core/templates/page.html

```diff
@@ -97,34 +109,35 @@
     </span><!-- /control -->
     <span class="control">
       <span class="glyphicon glyphicon-scissors" aria-hidden="true"></span>
-      <a id="clip" href="#" target="print">Clip Image</a>
+      <a id="clip" href="#" target="print">Clip/Print Image</a>
     </span><!-- /control -->
     {% endif %}
   </span><!-- /other_controls -->
 </div><!-- /imageViewer_nav -->
 
 
+{% if page.jp2_filename %}
 
-   
-      
-      {% if page.jp2_filename %}
-      
-      <div id="viewer_container_container">
+<div id="viewer_container_container">
```

Comment:
If you're back in this file, can you respace it to make sure that everything's lined up?  I had to read this a few times to realize that the div was inside the if, partly because I suck at reading but partly because of the spacing, haha.